agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -2147,7 +2157,17 @@ bool slave_thread::handle_cmdu_ap_manager_message(Socket *sd,
 
         send_cmdu_to_controller(cmdu_tx);
 
-        break;
+        beerocks_message::sClientMonitoringParams monitoring_params;
```

Comment:
Since you made start_monitoring a class method, all this can and should be done inside the function, to keep this switch case even shorter.